ste/JobPartPlan.go

```diff
@@ -45,6 +45,7 @@ type JobPartPlanHeader struct {
 	SourceRootLength       uint16            // The length of the source root path
 	SourceRoot             [1000]byte        // The root directory of the source
 	SourceExtraQueryLength uint16
+	SourceVersionIDs       []string
```

Comment:
>[]string [](start = 24, length = 8)

as we said, array of string cannot be used here. Make the version part of each transfer instead.

Reply:
Done